plugins/node/opentelemetry-instrumentation-redis-4/test/redis.test.ts

```diff
@@ -183,6 +185,54 @@ describe('redis@^4.0.0', () => {
     });
   });
 
+  describe('client connect', () => {
+    it('produces a span', async () => {
+      const client = createClient({
```

Comment:
I suggest giving it another name. `client` is already defined in the global `describe` scope which can be confusing to read.

Reply:
Renamed the connection test clients